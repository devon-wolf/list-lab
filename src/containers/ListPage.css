.listPage {
	background-color: palevioletred;
	display: flex;
	flex-direction: column;
	align-items: center;
	justify-content: center;
}