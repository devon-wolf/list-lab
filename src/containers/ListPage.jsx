import React, { Component } from 'react';
import style from './ListPage.css';
import VillagerList from '../components/displays/VillagerList';

export default class ListPage extends Component {
	render() {
		return (
			<div className={style.listPage}>
				<h2>Villagers</h2>
				<VillagerList 
					villagers={[
						{
							id: 1,
							name: 'test',
							image: 'fake',
							phrase: 'sure'
						}
					]}
				/>
			</div>
		)
	}
}
