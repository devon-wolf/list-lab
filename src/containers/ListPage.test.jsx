import React from 'react';
import { render, screen } from '@testing-library/react';
import ListPage from './ListPage';

describe('List page', () => {

	it('renders the list page', async () => {
		render(<ListPage />);

		const ul = await screen.findByRole('list', { name: 'villagers' });

		expect(ul).toBeInTheDocument();
	});

})