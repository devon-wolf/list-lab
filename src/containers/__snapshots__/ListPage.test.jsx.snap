// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List page renders the list page 1`] = `
<ul
  aria-label="villagers"
>
  <li>
    <h3>
      test
    </h3>
    <img
      alt="test"
      src="fake"
    />
    <p>
      sure
    </p>
  </li>
</ul>
`;
